src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -786,24 +869,24 @@ public void removeInstancePublicIp(Infrastructure infrastructure, String instanc
                                                                          .stream()
                                                                          .map(networkInterfaceId -> azureService.networkInterfaces()
                                                                                                                 .getById(networkInterfaceId))
-                                                                         .filter(networkInterface -> Optional.ofNullable(networkInterface.primaryIpConfiguration()
-                                                                                                                                         .getPublicIpAddress())
+                                                                         .filter(networkInterface -> Optional.ofNullable(networkInterface.primaryIPConfiguration()
+                                                                                                                                         .getPublicIPAddress())
                                                                                                              .isPresent())
                                                                          .filter(networkInterface -> !networkInterface.id()
-                                                                                                                      .equals(vm.getPrimaryPublicIpAddressId()))
+                                                                                                                      .equals(vm.getPrimaryPublicIPAddressId()))
                                                                          .findAny();
         if (optionalSecondaryNetworkInterface.isPresent()) {
-            PublicIpAddress publicIPAddress = optionalSecondaryNetworkInterface.get()
-                                                                               .primaryIpConfiguration()
-                                                                               .getPublicIpAddress();
-            optionalSecondaryNetworkInterface.get().update().withoutPrimaryPublicIpAddress().apply();
-            azureService.publicIpAddresses().deleteById(publicIPAddress.id());
+            PublicIPAddress PublicIPAddress = optionalSecondaryNetworkInterface.get()
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)